app/javascript/components/analytics/widgets/Demographics.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
 import { Card, Col, Row } from 'react-bootstrap';
-import { ResponsiveContainer, BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend } from 'recharts';
+import { Bar, BarChart, CartesianGrid, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
```

Comment:
Only added `import { mapToChartFormat, parseOutFields } from '../../../lib/AnalyticsUtils';`.  Everything else here is just moving `imports` around / alphabetizing them.